draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -651,6 +683,26 @@ cookie        = cookie-string
 cookie-string = cookie-pair *( ";" SP cookie-pair )
 ~~~
 
+While {{Section 5.4 of RFC9110}} does not define a length limit for header
+fields it is likely that the web server's implementation does impose a limit;
+many popular implementations have default limits of 8K. Servers SHOULD avoid
+setting a large number of large cookies such that the final cookie-string
+would exceed their header field limit. Not doing so could result in requests
+to the server failing.
+
+Servers MUST be tolerant of multiple cookie headers. For example, an HTTP/2
```

Comment:
Shouldn't we use 
```
`Cookie`
```
here to clarify what header we are talking about? That would also match Set-Cookie requirements above.

This applies quite a few times.